webview/src/experiments/components/table/header/TableHeader.tsx

```diff
@@ -27,7 +27,7 @@ const possibleOrders = {
 } as const
 
 interface TableHeaderProps {
```

Comment:
If I attempt to to hide a parent column that already had hidden children, it adds the missing children instead of hiding the entire column:

https://user-images.githubusercontent.com/43496356/197222423-89094ddc-b94c-41ab-a9cc-35936614a5c5.mov

Reply:
Sorry for the delay here, this is a tricky one

Reply:
I found the issue and added a (currently) failing test for it here: https://github.com/iterative/vscode-dvc/pull/2643/files#r1004679812

Reply:
I'll start fixing the issues there today